### DISCUSSION TITLE
How to translate third value

### DISCUSSION THREAD

**Discussion by kalwinskidawid** [livewire/livewire#5472] [1. Help]
I have a small problem with translation. 

I have function rules() which have two values to check:  
```php
public function rules()
    {
        return [
            'type' => ['required', 'string', 'in:basic,additional'],
            'cloth' => ['required_if:type,basic','nullable', 'string'],
        ];
    }
```
I have also function validationAttributes which: 
```php
 public function validationAttributes()
    {
        return [
            'type' => __('translate.type'),
            'cloth' => __('translate.cloth'),
            'basic' => __('translate.basic'),
            'additional' => __('translate.additional'),
        ];
    }
```
But when comes to return error from global translation:
`'required_if' => 'The :attribute field is required when :other is :value.',`

But when it comes to returning an error I got "The <translation_type> field is required when <translation_cloth> is basic"  
It looks like :attribute and :other are translated, byt the last value is not translated. How to fix it?

**Comment by joshhanley** [comment]
@kalwinskidawid hmm, maybe there is a bug, any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it? Thanks!
